### PR TITLE
85115 1010 ezr update submit error message

### PR DESCRIPTION
**Pull request by JoshingYou1**
## Summary

- Updates the content for the submission error message, which includes a link to the static page.

## Related issue(s)

- https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/85115

## Testing done

- This can be done in a review instance or staging: 
1. Log in as a staging user and fill out the EZR form. 
2. Once you reach the review page, right click in your browser window, select inspect, go to the "Application" tab, click on "Local storage", and click on the first item in the dropdown.
3. You should see a key named `crsfToken`. Double-click on the value and change it to something else.
4. Submit the form and you should see the updated submission error message.

## What areas of the site does it impact?

- 10-10EZR form

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/ezr/components/FormAlerts/SubmissionErrorAlert.jsx [hunk 1]
```diff
@@ -12,8 +12,13 @@ const SubmissionErrorAlert = () => {
       <va-alert status="error" uswds>
         <h3 slot="headline">{content['alert-submission-title']}</h3>
         <div>
-          <p>{content['alert-submission-message']}</p>
+          <p>{content['alert-submission-primary-message']}</p>
+          <p>{content['alert-submission-secondary-message']}</p>
         </div>
+        <a href="/health-care/update-health-information/#how-do-i-update-my-information">
+          Learn more about how to update your information by mail, phone, or in
```
Comment (coope93) [inline src/applications/ezr/components/FormAlerts/SubmissionErrorAlert.jsx:19]: Should this text be in the `content.json` file too?

**Review by coope93 — DISMISSED** [review]

**Review by rmessina1010 — DISMISSED** [review]
LGTM

File: src/applications/ezr/components/FormAlerts/SubmissionErrorAlert.jsx (hunk 1 above)
Comment (mkerns1) [reply src/applications/ezr/components/FormAlerts/SubmissionErrorAlert.jsx:19]: I'm not sure we want any of this in `content.json`. The pattern I've been following is that `content.json` is for use when we need a title or description inside a `uiSchema`. Anything inside of `components` doesn't need to go inside of `content.json`. At least that's the pattern I've been following.

Comment (mkerns1) [reply src/applications/ezr/components/FormAlerts/SubmissionErrorAlert.jsx:19]: Sorry disregard just realizing this isn't a new change to the component, just an update to the copy. Looks good to me!

**Review by mkerns1 — DISMISSED** [review]

**Review by mkerns1 — APPROVED** [review]
LGTM!

**Pull request merged**